himalaya.el

```diff
@@ -378,7 +381,7 @@ If ACCOUNT or MAILBOX are nil, use the defaults."
     (insert message)
     (set-buffer-modified-p nil)
     (himalaya-message-read-mode)
-    (goto-char (point-min))
+    (goto-line 1)
```

Comment:
Is there any reason you want to replace `goto-char` with `goto-line`? The function documentation for `goto-line` recommends using `(goto-char (point-min))` in most cases

Reply:
:heavy_check_mark: